### PR TITLE
Prepare release v2.0.0

### PR DESCRIPTION
**Pull request by sserrata**
## Description

See changelog for details

## Motivation and Context

First stable release for v2.0.0!

### CODE REVIEW

**Comment by sserrata** [comment]
Hi @blindaa121, so far so good. Appreciate a second pair of eyes on this release before we merge. Thanks!

**Review by blindaa121 — APPROVED** [review]
@sserrata Thanks for prepping the release! Everything looks good on my end 🚀

**Pull request merged**
